Structure_Engine/Query/Normal.cs

```diff
@@ -119,15 +119,16 @@ private static Vector Normal(this IAreaElement areaElement)
         }
 
         /***************************************************/
-        /**** Public Methods - ToBeRemoved               ****/
+        /**** Private Methods                           ****/
         /***************************************************/
 
         [ToBeRemoved("3.1", "ToBeRemoved by method targeting IElement2D.")]
         [Description("Returns the Panels local z-axis, a vector orthogonal to the plane of the Panel. This is found by fitting a plane through all the edge curves and taking the Normal from this plane.")]
         [Input("panel", "The Panel to evaluate the normal of.")]
         [Output("normal", "Vector representing the local z-axis Panel.")]
-        public static Vector Normal(this Panel panel)
+        private static Vector Normal(this Panel panel)
```

Comment:
I note the method has been made private but the `ToBeRemoved` attribute still exists, might be worth removing the attribute at some point @IsakNaslundBh ? (Maybe another, smaller PR that doesn't impact 10 others with checks?)

Reply:
ah, good catch! Think I just fix it in this PR for it not to be hanging again. Especially now that most alignment PRs already have been merged.